featureFlags/README.md

```diff
@@ -0,0 +1,164 @@
+# FeatureFlags
+
+# FeatureFlag object
+
+```
+{
+    'name': string,
+    'id': string
+    'title':string,
+    'created_at': timestamp,
+    'updated_at': timestamp,
+    'config': {'enabled': boolean},
+    'owner': string
+    'launched_at': timestamp
+}
+```
+
+## **Requests**
+
+|                 Route                 |             Description                  |
+|:-------------------------------------:|:----------------------------------------:|
+| [GET /featureFlags](#get-featureFlags)  |   Returns all feature flags in system  |
+| [POST /featureFlags](#post-featureFlags)   |   Creates a feature flag            |
+| [PATCH /featureFlags/:id](#patch-featureFlagsId)   | Updates data of featureflag |
+| [DELETE /featureFlags/:id](#delete-featureFlagsId) |  Deletes the feature flag   |
```

Comment:
`featureFlags`: the URLs are case-insensitive I suppose 🤔 
featureflags, FEATUREFLAGS, featureFlags are all the same, so it's better to go with all lower-case, I guess

Reply:
Yes. it would be better if we go with the lower-case letter!